parachain/pallets/ethereum-beacon-client/src/weights.rs

```diff
@@ -0,0 +1,48 @@
+//! Autogenerated weights for ethereum_beacon_client
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-09-27, STEPS: `10`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("/tmp/snowbridge/spec.json"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/snowbridge
+// benchmark
+// pallet
+// --chain
+// /tmp/snowbridge/spec.json
+// --execution=wasm
+// --pallet
+// ethereum_beacon_client
+// --extrinsic
+// *
+// --steps
+// 10
+// --repeat
+// 10
+// --output
+// pallets/ethereum-beacon-client/src/weights.rs
+// --template
+// module-weight-template.hbs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for ethereum_beacon_client.
+pub trait WeightInfo {	fn sync_committee_period_update() -> Weight;	fn import_finalized_header() -> Weight;	fn import_execution_header() -> Weight;}
+
+/// Weights for ethereum_beacon_client using the Snowbridge node and recommended hardware.
+pub struct SnowbridgeWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for SnowbridgeWeight<T> {	fn sync_committee_period_update() -> Weight {
+		(170_683_416_000 as Weight)			.saturating_add(T::DbWeight::get().reads(4 as Weight))			.saturating_add(T::DbWeight::get().writes(2 as Weight))	}	fn import_finalized_header() -> Weight {
```

Comment:
Are these weights generated using full mainnet config? Should make sure that we default to the most expensive weights, even if it isn't accurate for snowbase runtime

Reply:
Yes, they are generated using Snowblink using the mainnet config. I figured the same. 😄